src/nupic/regions/record_sensor.py

```diff
@@ -206,15 +206,13 @@ def getSpec(cls):
           accessMode="ReadWrite",
           count=1,
           constraints=""),
-        predictedFieldIdx=dict(
-          description="Index of the field to be predicted. Needs to be "
-                      "consistent with the data source indexing. "
-                      "Default value is < 0 which means that no particular "
-                      "field is selected. This will result in the outputs "
-                      "actValueOut and bucketIdxOut not being populated.",
-          dataType="UInt32",
+        predictedField=dict(
+          description="The field to be predicted. This will result in the "
+                      "outputs actValueOut and bucketIdxOut not being "
+                      "populated.",
+          dataType="Byte",
```

Comment:
The `dataType` is confusing because it's a string. Should the description say "The field _name_ to be predicted..."?